src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -49,4 +49,66 @@ class Wallet(
 
     return buildTransaction(sourceAddress, server, network, operations)
   }
+
+  // Add trustline
+  fun addAssetSupport(
+    sourceAddress: String,
+    assetCode: String,
+    assetIssuer: String,
+    trustLimit: String = "",
```

Comment:
It's more convenient to have a default value there, otherwise, it's required when `sponsorAddress` is provided (it makes sense to keep this after `trustLimit`). Maybe there is a trick in Kotlin to make that happen?

Your comment made me catch a bug for the default value, which cannot be an empty string. :sweat_smile:  We're now setting it to `Long.MAX_VALUE` (Int64). I also added a few more tests to catch these types of errors sooner. :crossed_fingers: